packages/client/hmi-client/src/services/charts.ts

```diff
@@ -456,22 +478,14 @@ function createStatisticLayer(
 			x: { field: options.timeField, type: 'quantitative', axis: xaxis },
 			y: { field: 'stat_value', type: 'quantitative', axis: yaxis },
 			color: {
-				field: 'stat_variable',
+				field: 'type',
 				type: 'nominal',
 				scale: {
-					domain: statisticalVariables,
-					range: [isPreStatistic ? '#AAB3C6' : '#1B8073']
+					domain: ['before', 'after'],
+					range: ['#AAB3C6', '#1B8073']
 				},
-				legend: options.legend
-					? {
-							title: null,
-							orient: 'top',
-							direction: 'horizontal',
-							labelExpr: isPreStatistic
-								? 'datum.value ? "Before Optimization" : ""'
-								: 'datum.value ? "After Optimization" : ""'
-						}
-					: null
+				// we only want to render a legend for the first statistic layer
```

Comment:
only render the first statistic layer legend so we can vertically align it.  the legend otherwise doesnt play nice...